spec/services/cleaners/advocate_claim_cleaner_spec.rb

```diff
@@ -12,102 +48,72 @@
   describe '#call' do
     subject(:call_cleaner) { cleaner.call }
 
-    context 'with an "Appeal against sentence" case type including a fixed fee' do
-      let(:claim) do
-        create(:draft_claim, case_type: create(:case_type, :appeal_against_sentence), fixed_fees: fixed_fees)
-      end
-      let(:fixed_fees) { [build(:fixed_fee, :fxase_fee, :with_date_attended, rate: 9.99)] }
+    let(:claim) do
+      build(
```

Comment:
does this work if you use `create` here too. Just thinking about hooks and their impact on all this

Reply:
The problem is that create would run the cleaner before the test, which is why the old versions of the tests set up the claim with one case type and then updated it.

Maybe something like this would work?

```ruby
let(:claim) do
  create(:advocate_final_claim, :draft, case_type: case_type).tap do |claim|
    claim.fixed_fees = [build(:fixed_fee, :fxase_fee, :with_date_attended, rate: 9.99)]
    # etc
  end
end
```